kevm-pyk/src/kevm_pyk/kproj/evm-semantics/lemmas/bitwise-simplification.k

```diff
@@ -60,39 +60,6 @@ module BITWISE-SIMPLIFICATION [symbolic]
        andBool 0 <=Int Y andBool Y <Int pow256 -Int X
       [simplification, concrete(X), preserves-definedness]
 
-    // Generalization of: maxUIntXXX &Int #asWord ( BA )
```

Comment:
These are subsumed by the slot update lemmas.

Reply:
It seems that deleting these lemmas is the bulk of this change. Does just deleting these lemmas, and commenting the handful of proofs, suffice for this PR? Or do we also need to move the slot update reasoning module stuff?

Reply:
It's a bit the other way round, will explain in PR info.